doc/tutorials/static-ip.rst

```diff
@@ -0,0 +1,67 @@
+.. _static_ip:
+
+Using a static IP address
+#########################
+
+When you install ACRN on your system following the :ref:`getting_started`, a
+bridge called ``acrn-br0`` will be created and attached to the Ethernet network
+interface of the platform. By default, the bridge gets its network configuration
+using DHCP. This guide will explain how to modify the system to use a static IP
+address. You need ``root`` privileges to make these changes to the system.
+
+ACRN Network Setup
+******************
+
+The ACRN Service OS is based on `Clear Linux`_ and it uses `systemd-networkd`_
+to set-up the Service OS networking. A few files are responsible for setting up the
+ACRN bridge (``acrn-br0``), the TAP device (``acrn_tap0``) and how these are all
+connected. Those files are installed in ``/usr/lib/systemd/network``
+on the target device and can also be found under ``tools/acrnbridge`` in the source code.
+
+Setting up the static IP address
+********************************
+
+In order to set-up a static IP address, you need to modify the
+``/usr/lib/systemd/network/50-eth.network`` file, and more specifically the ``[Network]``
+section. This is the content of the file used in ACRN by default.
+
+.. literalinclude:: ../../tools/acrnbridge/eth.network
+   :caption: tools/acrnbridge/eth.network
+   :emphasize-lines: 5
+
+Edit the file to remove the line highlighted above and add your network settings in
+that ``[Network]`` section. You will typically need to add the ``Address=``, ``Gateway=``
+and ``DNS=`` parameters in there. There are many more parameters that can be used and
+detailing them is beyond the scope of this document. For an extensive list of those,
+please visit the official `systemd-network`_ page.
+
+This is an example of what a typical ``[Network]`` section would look like:
+
+.. code-block:: none
+
+   [Network]
+   Address=192.168.1.87/24
+   Gateway=192.168.1.254
+   DNS=192.168.1.254
+
+Activate the new configuration
+******************************
+
+You do not need to reboot the machine after making the changes to the system, the
+following steps will suffice (run as ``root``).
```

Comment:
clarify with:

    following steps that restart the systemd-networkd daemon will suffice (run as ``root``):